tools/Microsoft.StandardsPolice/project.json

```diff
@@ -1,6 +1,10 @@
 ﻿{
   "version": "1.0.0-*",
 
+  "compilationOptions": {
```

Comment:
Building Microsoft.AspNet.Server.Kestrel for DNX,Version=v4.5.1
Error: Could not load file or assembly 'Microsoft.StandardsPolice, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. A strongly-named assembly is required. (Exception from HRESULT: 0x80131044)
fail: Exit code 1 from cmd